modules/300-prometheus/templates/alertmanager/internal/ingress.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if and (.Values.global.modules.publicDomainTemplate) (.Values.prometheus.internal.alertmanagers.internal) }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: alertmanager
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list . (dict "app" "alertmanager")) | nindent 2 }}
+  annotations:
+    web.deckhouse.io/export-name: "alertmanager"
```

Comment:
The icon is required for exposed web services.

Reply:
We agreed to use grafana for this purpose:
1. Deploy the dedicated ingress resource for each alermanager
2. Use annotations to configure path

Reply:
Icon added, ingresses deployed